parser/parser.go

```diff
@@ -715,7 +715,7 @@ func (p *parser) parseTerminal() (ast.Node, error) {
 		return &ast.LiteralString{
 			NodeBase:    ast.NewNodeBaseLoc(tok.loc),
 			Value:       tok.data,
-			Kind:        ast.StringDouble,
```

Comment:
Did this bug get missed by a test?

Reply:
This is pretty subtle, b/c I think we treat them exactly the same after parsing with one exception - imports. And there is no unparsing in Go version, so it didn't affect the behavior until now.